hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -864,6 +866,9 @@ private void startScheduler() throws Exception {
             schedulerConfig.setMetricDispatcherMaxBatchSize(this.configuration.getMetricDispatcherMaxBatchSize());
             schedulerConfig.setAvailDispatcherBufferSize(this.configuration.getAvailDispatcherBufferSize());
             schedulerConfig.setAvailDispatcherMaxBatchSize(this.configuration.getAvailDispatcherMaxBatchSize());
+            schedulerConfig.setPingDispatcherPeriodSeconds(this.configuration.getPingDispatcherPeriodSeconds());
+            schedulerConfig.setPingDispatcherFeedId(this.feedId);
```

Comment:
why don't we just make this "setFeedId" since its possible in the future other schedulers might need to know the agent's feed ID. Change the getter too to be "getFeedId"